### PR TITLE
Use Throttle component

### PR DESCRIPTION
**Pull request by craig1123**

### CODE REVIEW

File: __tests__/useThrottle.test.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+describe('useThrottle', () => {
+  beforeAll(() => {
+    jest.useFakeTimers();
+  });
+
+  afterEach(() => {
+    jest.clearAllTimers();
+  });
+
+  // afterAll(() => {
```
Comment (madrich27times) [inline __tests__/useThrottle.test.js]: Delete this?

File: README.md [hunk 2]
```diff
@@ -44,6 +44,7 @@ yarn add react-recipes
 | 🥑 [`useOnClickOutside`](#-useOnClickOutside)   | -                                                     | (ref, callback)                                                                         |
```
Comment (madrich27times) [inline README.md]: Did you still want to change this emoji? We could do the orange here

File: __tests__/useThrottle.test.js (hunk 1 above)
Comment (craig1123) [reply __tests__/useThrottle.test.js]: I put it back in

File: README.md (hunk 2 above)
Comment (craig1123) [reply README.md]: Let's review the emojis after I add more recipes (I have a list of ~10+)

**Pull request merged**